db/seeds/dev/music.js

```diff
@@ -0,0 +1,64 @@
+const getAllData = require('../../../itunesCall.js');
+
+const createMusic = async (knex, music) => {
+  console.log(music)
+  const newmusic = await knex('music').insert(
+    {
+      artist: music.artistName,
+      genre: music.primaryGenreName,
+      song_title: music.trackName,
+      album_title:music.collectionName,
+      album_url: music.collectionViewUrl,
+      release_date: music.releaseDate,
+      type: music.trackExplicitness,
+      image_60: music.artworkUrl60,
+      image_100:music.artworkUrl100,
+    });
+}
+
+exports.seed = async (knex) => {
+  try {
+    await knex('music').del()
+    let musicPromise = await getAllData('music')
+    let musicTableData = musicPromise.results.map(music => {
+      return createMusic(knex, music);
+    });
+    return Promise.all(musicTableData);
+  } catch (error) {
+    console.log(`Error seeding data: ${error}`)
+  }
+};
+
+// {
+//   wrapperType: 'track',
```

Comment:
Is there a reason to keep this commented out code in here?  Not sure if you want to keep it to refer to it later, but I think it would be a good idea to save this code somewhere else and remove it from this file.

Reply:
I just used it when assigned the values to the columns in the music table, I forgot to remove them.